cloudant.js

```diff
@@ -233,55 +209,11 @@ function Cloudant(options, callback) {
   nano.delete_virtual_host = delete_virtual_host; // eslint-disable-line camelcase
 
   if (callback) {
-    debug('Automatic ping');
-    nano.ping(login, function(er, pong, cookie) {
-      if (er) {
-        callback(er);
-      } else {
-        if (cookie) {
-          requestDefaults.headers.cookie = cookie;
-        }
-
-        callback(null, nano, pong);
-      }
+    nano.cc.addPlugins('cookieauth');
```

Comment:
So if I'm understanding this correctly, if you request the ping callback then the `cookieauth` plugin is enabled by default? I thought we were going to turn `cookieauth` on by default unless a plugin list was supplied that explicitly removed it?

Reply:
Yeah, the ping callback enables cookie auth. I don't like it much. But it's following the old pattern.

I haven't turned cookie auth on by default yet. It's something we can do. I wasn't too sure how a client would go about disabling it though. The cloudant client is exposed so I could add a new `removePlugins` method (to mirror `addPlugins`), i.e.
```js
const client = Cloudant({ plugins: ['retry'] });
client.cc.removePlugins('cookieauth');
```
I don't want to clutter this pr even more so I'll open a new one for this.